### PR TITLE
bugsnag: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/bugsnag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3
+    - setuptools
```
Comment (scopatz) [inline recipes/bugsnag/meta.yaml]: This probably doesn't need pip and setuptools

Comment (scopatz) [reply recipes/bugsnag/meta.yaml]: ie you can get rid of setuptools

Comment (tschoonj) [reply recipes/bugsnag/meta.yaml]: Setup.py uses setuptools though

Comment (scopatz) [reply recipes/bugsnag/meta.yaml]: Yep, but setuptools is an implicit dep of pip

Comment (tschoonj) [reply recipes/bugsnag/meta.yaml]: setuptools dependency removed

**Comment by tschoonj** [comment]
Many thanks!

By the way: if you have a moment, could you have look at https://github.com/conda-forge/ispyb-feedstock? I tried making a python 3.8 build there but conda smithy rerender didn't seem to work... any suggestions?

**Comment by scopatz** [comment]
It is generally best to wait for the bot to put in a 3.8 update PR, so that all of the dependencies are available first. you can go to http://conda-forge.org/status to see where the package is in this migration

**Comment by tschoonj** [comment]
The ispyb [recipe](https://github.com/conda-forge/ispyb-feedstock/blob/master/recipe/meta.yaml) is pretty simple, it's only real python dependency is `mysql-connector-python`, which has python 3.8 builds already, so I don't understand why the update PR was never opened.

**Comment by scopatz** [comment]
probably best to ask at https://gitter.im/conda-forge/regro-cf-autotick-bot

**Pull request merged**